README.md

```diff
@@ -157,4 +157,28 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](https://github.com/MosDevx/MovieApiProject/issues).
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please star it ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
Hi @MosDevx
Good job so far!

To Highlight! 👏 🟢

Good commit messages,
Great folder structure
Clean code
Good PR description
Your project is running free of Linter errors
There is some issue that you still need to work on but you are almost there!

Required Changes ♻️
- Kindly remove the same element descriptions on your README file
- Kindly add the link of your project live demo